src/app/portlets/dot-edit-page/layout/components/dot-theme-selector/dot-theme-selector.component.ts

```diff
@@ -39,7 +42,19 @@ export class DotThemeSelectorComponent implements OnInit {
                 'dot.common.apply',
                 'dot.common.cancel'
             ])
-            .subscribe();
+            .subscribe(() => {
+                this.closeDialogAction = {
+                    label: this.dotMessageService.get('dot.common.cancel'),
+                    action: (dialogElem => { dialogElem.closeDialog(); })
```

Comment:
Why don't we make the `dot-dialog` that by default the cancel button close the dialog? that way we don't have to close it ourselves in every place we use it.